src/pipeline_blocks.py

```diff
@@ -470,6 +532,51 @@ def stacking_features(config, train_mode, suffix, **kwargs):
         return feature_combiner
 
 
+def stacking_normalization(features, config, train_mode, suffix, **kwargs):
```

Comment:
@kstrzala Since we are working on predictions at this point I am wondering which (if any) normalization strategy should one choose. I guess for our problem normalized rank could be a better choice than normalizing the predictions. What do you think?